src/aws-cpp-sdk-core/include/aws/core/client/ClientConfiguration.h

```diff
@@ -440,7 +440,22 @@ namespace Aws
                */
               ResponseChecksumValidation responseChecksumValidation = ResponseChecksumValidation::WHEN_SUPPORTED;
             } checksumConfig;
-
+            
```

Comment:
i think you moved this to `credentialProviderConfig`, so this doesnt need to exist anymore?